cicd/pipelines/securing-webhooks-with-event-listeners.adoc

```diff
@@ -0,0 +1,22 @@
+[id="securing-webhooks-with-event-listeners"]
+= Securing webhooks with event listeners
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+
+:context: securing-webhooks-with-event-listeners
+
+toc::[]
+
+Triggers in {pipelines-title} support both HTTP (insecure) and HTTPS (secure HTTP) connections to the `Eventlistener` resource. With the secure HTTPS connection, you get an end-to-end secure connection within and outside the cluster. After you create a namespace, you can enable this secure HTTPS connection for the `Eventlistener` resource by adding the `operator.tekton.dev/enable-annotation=enabled` label to the namespace, and then creating a `Trigger` resource and a secured route using re-encrypt TLS termination.
+
+When an OpenShift pipeline is installed through an operator a `tekton-operator-proxy-webhook-` pod will always be running. The webhook watches for labels on the namespace and the below annotation is set on the `Eventlistener` object, whenever one is found, which in turn creates secrets and required certificates:
+
+[source,terminal]
+----
+service.beta.openshift.io/serving-cert-secret-name=<secret_name>
+----
+
+You can further mount the created secret into the `Eventlistener` pod to secure the request.
```

Comment:
In addition, you can mount the created secret into the `Eventlistener` pod to secure the request.